evm/src/cpu/kernel/asm/core/util.asm

```diff
@@ -1,13 +1,10 @@
 // Return the next context ID, and record the old context ID in the new one's
 // @CTX_METADATA_PARENT_CONTEXT field. Does not actually enter the new context.
 %macro create_context
+    // stack: (empty)
     %next_context_id
-    GET_CONTEXT
-    %stack (ctx, next_ctx)
-       -> (next_ctx, @SEGMENT_NORMALIZED_TXN, @CTX_METADATA_PARENT_CONTEXT,
```

Comment:
Not sure why the segment was `SEGMENT_NORMALIZED_TXN` here (should be `SEGMENT_CONTEXT_METADATA` right?), but this block is exactly the same as `%set_new_ctx_parent_ctx`.